### PR TITLE
Adding custom move feedrate for G26

### PR DESCRIPTION
**Pull request by ScrewThisBanana**
### Requirements

`G26_MESH_VALIDATION` enabled in configuration, no additional requirements

### Description

This commit adds an additional configuration parameter (`G26_XY_FEEDRATE_MOVE`) in the Configuration.h file that can be used to specify the movement speed during the G26 validation pattern command during moves without extrusion.

The added feature was tested on my personal printer (TwoTrees Sapphire Pro) and works like a charm. Only x/y movements without e components take the newly introduced feedrate. All other behaviours of the G26 command stay the same.

### Benefits

The amount of time used for G26 is drastically reduced as the feedrate for print moves is now different from moves without E-components

### Configurations

[platformio.ini.txt](https://github.com/MarlinFirmware/Marlin/files/5791247/platformio.ini.txt)
[Configuration.h.txt](https://github.com/MarlinFirmware/Marlin/files/5791251/Configuration.h.txt)

### Related Issues

Closes MarlinFirmware/Marlin#20615

### CODE REVIEW

**Comment by qwewer0** [comment]
I think `G26_XY_FEEDRATE_TRAVEL` would reflect the setting more, and maybe `100` would be safer, while still acceptable as default value.
@ScrewThisBanana What do you think?

**Comment by qwewer0** [comment]
Tried it out, works as expected.

**Comment by ScrewThisBanana** [comment]
> I think `G26_XY_FEEDRATE_TRAVEL` would reflect the setting more, and maybe `100` would be safer, while still acceptable as default value.
> @ScrewThisBanana What do you think?

I will change that two topics tomorrow

**Comment by ScrewThisBanana** [comment]
> > I think `G26_XY_FEEDRATE_TRAVEL` would reflect the setting more, and maybe `100` would be safer, while still acceptable as default value.
> > @ScrewThisBanana What do you think?
> 
> I will change that two topics tomorrow

"Tomorrow" is relative - fixed now ^^ See https://github.com/MarlinFirmware/Marlin/pull/20729/commits/2344ac46d94df0174a7018f0b120699f643ca153

File: Marlin/Configuration.h [hunk 1]
```diff
@@ -1361,6 +1361,7 @@
     #define MESH_TEST_HOTEND_TEMP  205    // (°C) Default nozzle temperature for the G26 Mesh Validation Tool.
     #define MESH_TEST_BED_TEMP      60    // (°C) Default bed temperature for the G26 Mesh Validation Tool.
     #define G26_XY_FEEDRATE         20    // (mm/s) Feedrate for XY Moves for the G26 Mesh Validation Tool.
+    #define G26_XY_FEEDRATE_TRAVEL    150   // (mm/s) Feedrate for XY Moves without extrusion for the G26 Mesh Validation Tool
```
Comment (qwewer0) [inline Marlin/Configuration.h]: @ScrewThisBanana Could you align it, like the values above?

Comment (ScrewThisBanana) [reply Marlin/Configuration.h]: Ofc

Comment (ScrewThisBanana) [reply Marlin/Configuration.h]: @qwewer0 done: https://github.com/MarlinFirmware/Marlin/pull/20729/commits/aaa873e4eb6a13d09f16a118d0da551a3f558a00

File: Marlin/Configuration.h [hunk 2]
```diff
@@ -1361,6 +1361,7 @@
     #define MESH_TEST_HOTEND_TEMP  205    // (°C) Default nozzle temperature for the G26 Mesh Validation Tool.
     #define MESH_TEST_BED_TEMP      60    // (°C) Default bed temperature for the G26 Mesh Validation Tool.
     #define G26_XY_FEEDRATE         20    // (mm/s) Feedrate for XY Moves for the G26 Mesh Validation Tool.
+    #define G26_XY_FEEDRATE_TRAVEL  100   // (mm/s) Feedrate for XY Moves without extrusion for the G26 Mesh Validation Tool
```
Comment (qwewer0) [inline Marlin/Configuration.h:1364]: Decimal point should be aligned for all.

**Comment by Roxy-3D** [comment]
@ScrewThisBanana    This is a long overdue improvement.   

Thank You!

**Comment by Roxy-3D** [comment]
@ScrewThisBanana    Ooooops!   I shouldn't have merged this yet.   We need to keep all the example Configuration.h files synchronized with the default Configuration.h file.     I need to revert this.

As soon as the Example Configuration.h files have the new option in them we can continue.

UPDATE:   I reverted the changes.    I have very little time right now, but over the next couple of days I'll try to create a new Pull Request with your changes and the option added to all of the Configuration.h files.   (And at that point it can be merged.)    Or as an alternative, you can create a new Pull Request with all of the desired changes.   Either path works for me.

**Comment by ScrewThisBanana** [comment]
@Roxy-3D how should I proceed? Pull example configuration files from https://github.com/MarlinFirmware/Configurations and add the new configuration parameter in each example configuration file?

**Comment by Roxy-3D** [comment]
@ScrewThisBanana     I apologize!   I seem to be a little bit out of the loop.   What I said used to be very true.   All of the (many) Example Configuration.h files needed to be updated to reflect any new options added.    But that is no longer true.

What you had was sufficient and I should not have reverted the merge.     I don't think there is an easy way for me to recover the original Pull Request.   If you can start a fresh Pull Request with the exact changes reflected up above, I will get it merged again.

**Comment by ScrewThisBanana** [comment]
@Roxy-3D A new pull request (https://github.com/MarlinFirmware/Marlin/pull/20879) was opened, feel free to try again ;)

File: Marlin/Configuration.h (hunk 2 above)
Comment (ScrewThisBanana) [reply Marlin/Configuration.h:1364]: By the way, your change regarding the decimal point alignment is done in the new (needed) PR https://github.com/MarlinFirmware/Marlin/pull/20879 in commit https://github.com/MarlinFirmware/Marlin/pull/20879/commits/5d930a35753b399ba5234cbb19a77f29dde10858 - I finally understood what you wanted from me ;) I hope that is ok for you?

Comment (qwewer0) [reply Marlin/Configuration.h:1364]: All ok, thank you.

**Comment by qwewer0** [comment]
@Roxy-3D Can you remove/delete [revert-20729-bugfix-2.0.x branch](https://github.com/MarlinFirmware/Marlin/tree/revert-20729-bugfix-2.0.x)? I think it served it's purpose and will only cause confusion.

**Pull request merged**